### DISCUSSION TITLE
Installation on Linux or Windows

### DISCUSSION THREAD

**Discussion by adsoink** [tailpress/tailpress#212] [Q&A]
Install instructions appear to be Mac-only and antiquated. Any chance of updating the readme with instructions for additional operating systems? If no, can the readme be updated with system requirements?

**Comment by jeffreyvr** [comment]
Why antiquated?

I use MacOS myself, but other people have been using the installer with Windows as well. So if there anything specific that is not working, is outdated or needs more info - feel free to contribute by replying here or submitting a PR.
